test/CruzoMarket.ts

```diff
@@ -20,6 +22,8 @@ describe("CruzoMarket", () => {
 
   const serviceFee = 300;
   const serviceFeeBase = 10000;
+  const rawVaultFuncSignature = RAW_VAULT_FUNCTION_SIGNATURE
```

Comment:
Meaningless assignment, you can just use the imported RAW_VAULT_FUNCTION_SIGNATURE value directly